upcase_booklist_app/templates/authors.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Curious Programmer</title>
+        <link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+
+    <body>
+        <div class="page-header">
+            <p class="nav-item">
+                <a href="{{ url_for('book_list') }}">Books</a>
+                <a href="{{ url_for('authors') }}">Authors</a>
+            </p>
+        </div>
+
+        <div class="page-title">
+            <h1>Authors</h1>
+        </div>
+
+        <div>
+            {% for book, details in books.items() %}
+                <p>
+                    <a href="{{ url_for('single_book', book_id=book) }}">
```

Comment:
So the variable `book` is misleading; this is really the `book_id`, not the `book` itself. It can be confusing otherwise.